apps/vessel-history/features/profile/Profile.tsx

```diff
@@ -147,6 +154,7 @@ const Profile: React.FC = (props): React.ReactElement => {
       dispatch(setHighlightedEvent(undefined))
       dispatch(setVoyageTime(undefined))
     }
+    updateDataview()
```

Comment:
Why not verifying that the `vessel.dataview.id` does not match the current vessel id before calling `updateDataview()`? I think that would be convenient to prevent updating many times the vessel dataview

Reply:
was my first try just to test and worked so good, and that check that you mention looks that is only to avoid the case a uses exit a profile and join to the same profile, but as all is in the store there is no performance issues or delays @rdgfuentes

Reply:
you are right, I dont see problems but it is not ok